tests/unit-core/test-container.c

```diff
@@ -122,6 +167,73 @@ main (void)
   jerry_gc (JERRY_GC_PRESSURE_LOW);
   TEST_ASSERT (global_counter == 1);
 
+  run_eval ("function assert (v) {\n"
+            "  if (v !== true)\n"
+            "    throw 'Assertion failed!'\n"
+            "}");
+
+  run_eval ("function test_values (arr1, arr2) {\n"
+            "  assert (arr1.length == arr2.length);\n"
```

Comment:
Use `===`. Furthermore, since this code requires ESNEXT, you can also use `Array.isArray()` to check that arr1 is an Array.